packages/shared-ux/file/file_upload/impl/src/file_upload.component.tsx

```diff
@@ -79,12 +77,14 @@ export const FileUpload = React.forwardRef<EuiFilePickerClass, Props>(
     const id = useGeneratedHtmlId({ prefix: 'filesFileUpload' });
     const errorId = `${id}_error`;
 
+    const formMaxWidth = mathWithUnits(euiTheme.size.base, (x) => x * 25);
```

Comment:
This copies the `euiFormMaxWidth` function in EUI: https://github.com/elastic/eui/blob/7ec3fafaac4d6a31d5d324a118c5cb98b7395922/packages/eui/src/components/form/form.styles.ts#L28

@elastic/eui-team is there a way we can access this through the EUI context?

Reply:
ℹ️ We're introducing a new token for it `euiTheme.components.forms.maxWidth` ([PR](https://github.com/elastic/eui/pull/8221)).